comment/comment.go

```diff
@@ -89,6 +90,24 @@ func (m *GormCommentRepository) Save(ctx context.Context, comment *Comment) (*Co
 	return comment, nil
 }
 
+// Delete a single comment
+func (m *GormCommentRepository) Delete(ctx context.Context, comment *Comment) error {
+	c := Comment{}
+	tx := m.db.Where("id=?", comment.ID).First(&c)
+	if tx.RecordNotFound() {
```

Comment:
It's generally a bit silly to have a Object as an argument, then fetch it from the database, for so to delete it. I'm sure we're doing it in many places, but... 

if Delete(comment).RowsEffected == 0 {
   return NotFound
}

Would do the job just fine.